docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
+
+Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 or less.
+
+To do this, we’re going to need an ‘ineligible.html’ page, and some logic to decide when to send them there.
+
+This kind of logic goes in a file called `app/routes.js`. Routes tell the kit what to do when the user goes to specific pages, and it’s written in JavaScript.
+
+The route takes the answer the user gave to the first question and either shows them the next question or the ineligible page, depending on their answer.
+
+## Create the ineligible page
+
+1. Make an `ineligible.html` page by copying `template-content-page.html`
+
+2. Update the content to tell the user why they’re ineligible and what they can do next.
+
+Check it works by visiting http://localhost:3000/ineligible
+
+## Create a route
+
+We’re going to write some logic to process the user’s answer to question 1. If the user can juggle 3 balls or more, we’ll show them question 2. If they answer with anything else, we’ll show them the ineligible page.
+
+Currently, the `juggling-balls` page sends the user directly to question 2. We’re going to instead send them to a new special url where we can run some code to check their answer.
```

Comment:
Could delete "new" unless it tells users something that helps them achieve their aim.